libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -1274,7 +1274,6 @@ string YulUtilFunctions::storageArrayPushFunction(ArrayType const& _type)
 {
 	solAssert(_type.location() == DataLocation::Storage, "");
 	solAssert(_type.isDynamicallySized(), "");
-	solUnimplementedAssert(_type.baseType()->storageBytes() <= 32, "Base type is not yet implemented.");
```

Comment:
I don't know what check this was supposed to be... ;)

Reply:
I don't know as well, very helpful message in assert as well :)